### PR TITLE
Minor css fixes, es6 refactoring

### PR DESCRIPTION
**Pull request by Archaeologist03**
Signed-off-by: $ Ilija <omniiknight@hotmail.com>

### CODE REVIEW

File: website/static/js/code-blocks-buttons.js [hunk 1]
```diff
@@ -31,18 +29,14 @@ window.addEventListener('load', function() {
   );
 
   const clipboard = new ClipboardJS('.btnClipboard', {
-    target: function(trigger) {
-      return trigger.parentNode.querySelector('code');
-    },
+    target: (trigger) => trigger.parentNode.querySelector('code')
   });
 
   clipboard.on('success', function(event) {
```
Comment (aswathkk) [inline website/static/js/code-blocks-buttons.js]: It would be better if you could use arrow function here.

File: website/static/css/code-blocks-buttons.css [hunk 2]
```diff
@@ -22,7 +22,8 @@ pre .btnIcon:hover {
 
 .btnIcon__body {
   align-items: center;
-  display: flex;
+  display: -webkit-flex;
+  display: flex;
```
Comment (sagarkrsd) [inline website/static/css/code-blocks-buttons.css]: This will override the first one so please declare only one of them !

File: website/static/css/custom.css [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 @media (max-width: 375px) {
   .logo {
+    display: -webkit-flex;
```
Comment (sagarkrsd) [inline website/static/css/custom.css]: Same here, second display field value will override this one !

File: website/static/css/custom.css [hunk 4]
```diff
@@ -289,7 +290,7 @@ and (max-device-width : 720px) {
 
 /* ReDoc doesn't change outer page styles */
 .redoc-body{
-    margin: 0px;
+    margin: 0;
```
Comment (sagarkrsd) [inline website/static/css/custom.css]: Why this change ?
